package.json

```diff
@@ -19,7 +19,7 @@
   "main": "./lib",
   "dependencies": {
     "buffer-writer": "1.0.1",
-    "generic-pool": "2.1.1",
+    "generic-pool": "git://github.com/Shyp/node-pool.git#shyp-master",
```

Comment:
I'm not opposed, and I agree that you shouldn't rely on github for installs. I'm not sure when it's going to get merged, and I don't want to put any pressure on the maintainer. I wouldn't mind publishing this on a fork, but I'm also a little wary about contributing to package bloat :(